app/org/maproulette/models/dal/TaskDAL.scala

```diff
@@ -645,10 +645,12 @@ class TaskDAL @Inject()(override val db: Database,
     if (reviewNeeded) {
       this.cacheManager.withOptionCaching { () => Some(task.copy(status = Some(status),
                                                  reviewStatus = Some(Task.REVIEW_STATUS_REQUESTED),
-                                                 reviewRequestedBy = Some(user.id))) }
+                                                 reviewRequestedBy = Some(user.id),
+                                                 modified = new DateTime())) }
```

Comment:
This should be handled by a trigger in the database. So if it isn't working we need to check there.

Reply:
The database is correctly updating the modified timestamp. The problem was that a the cache object being returned did not have the changed field -- so this change only updates the modified date on the cache object so that what's immediately returned makes sense. Next time that cache object is refreshed it will get the value from the database.